src/postgraphile/http/subscriptions.ts

```diff
@@ -88,6 +88,15 @@ export async function enhanceHttpServerWithWebSockets<
 
   const contextKey = (ws: WebSocket, opId: string): string => ws['postgraphileId'] + '|' + opId;
 
+  const releaseAllContextsForSocket = (ws: WebSocket): void => {
```

Comment:
```suggestion

  // IMPORTANT: if you change this, be sure to change `releaseAllContextsForSocket` too
  const contextKey = (ws: WebSocket, opId: string): string => ws['postgraphileId'] + '|' + opId;

  const releaseAllContextsForSocket = (ws: WebSocket): void => {
```

Reply:
@benjie just a ping - you've merged the PR before merging this suggestion, guess you could commit it directly on `master`.

Reply:
Yeah PR incoming :)

Reply:
https://github.com/graphile/postgraphile/pull/1613